etc/openqa/openqa.ini

```diff
@@ -71,6 +71,10 @@
 ## Gracefully restart the prefork workers if they reach a certain memory limit (in kB)
 #max_rss_limit = 180000
 
+## Causes jobs reported as incomplete by the worker to be cloned automatically when the
```

Comment:
I suggest you also describe the feature in documentation, e.g. similar to how I have done in https://github.com/os-autoinst/openQA/pull/3603/files#diff-71734bf4d91118f92743c7d0db9218c4f24729410e020110884606b5ad6a3878R751 . Could be just the next section after that

Reply:
added